packages/vscode-extension/src/util/DateUtils.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+export function getTimestamp(): string {
+    const dateNow = new Date();
+    const datePartsArray = [dateNow.getFullYear(), dateNow.getMonth() + 1, dateNow.getDate()].map(date => date.toString().length === 1 ? "0" + date : date);
+    const timePartsArray = [dateNow.getHours(), dateNow.getMinutes(), dateNow.getSeconds()].map(time => time.toString().length === 1 ? "0" + time : time);
+
+    return datePartsArray.join("/") + " " + timePartsArray.join(":");
```

Comment:
I like `/` but maybe we should use `-` to join the date parts as in story description.

Reply:
Thanks for noticing this. I was so used to using "/" for dates that I didn't notice the story mentioned to make use of "-". I updated this.